notebook/static-src/edit/js/editor.js

```diff
@@ -1,26 +1,20 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-define([
-    'jquery',
-    'base/js/utils',
-    'codemirror/lib/codemirror',
-    'codemirror/mode/meta',
-    'codemirror/addon/comment/comment',
-    'codemirror/addon/dialog/dialog',
-    'codemirror/addon/edit/closebrackets',
-    'codemirror/addon/edit/matchbrackets',
-    'codemirror/addon/search/searchcursor',
-    'codemirror/addon/search/search',
-    'codemirror/keymap/emacs',
-    'codemirror/keymap/sublime',
-    'codemirror/keymap/vim',
-    ],
-function($,
-    utils,
-    CodeMirror
-) {
     "use strict";
+        
+    var CodeMirror = require('codemirror/lib/codemirror');
+    var utils = require('base/js/utils');
+    require('codemirror/mode/meta');
+    require('codemirror/addon/comment/comment');
+    require('codemirror/addon/dialog/dialog');
+    require('codemirror/addon/edit/closebrackets');
+    require('codemirror/addon/edit/matchbrackets');
+    require('codemirror/addon/search/searchcursor');
+    require('codemirror/addon/search/search');
+    require('codemirror/keymap/emacs');
+    require('codemirror/keymap/sublime');
+    require('codemirror/keymap/vim');
```

Comment:
In utils.js CodeMirror was loaded using requirejs whereas here CodeMirror is loaded via node require. What's the reason for the difference?